Automattic/wp-calypso
# Gutenboarding: stop existing users A/B test

#### Changes proposed in this Pull Request
* stop existingUsersGutenbergOnboard test
* cleanup Gutenboarding test code from abtest config and signup controller 

#### Testing instructions
* Go to https://calypso.live/?branch=update/remove-gutenboarding-abtest
* Open console and run `localStorage.setItem( 'ABTests', '{"existingUsersGutenbergOnboard_20200911":"gutenberg"}' )`
* Go to `/start` => no redirect should happen
